lib/Markup/Render/MarkupJekyll.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+
+public struct JekyllGenerator: Render {
+    let permalink: String
+    
+    public init(permalink: String) {
+        self.permalink = permalink
+    }
+
+    public func render(content: String) -> String? {
+        guard let syntaxTree = SyntaxAnalyzer.parse(content: content) else { return nil }
+        let filteredSyntaxTree = syntaxTree.filter { !$0.isHidden }.reduce()
+        //if VERBOSE { syntax.forEach { print($0) } }
```

Comment:
Remove commented line.

Reply:
I was thinking to allow it as argument from the command line...but yes, we have to remove it